local/php/envs.go

```diff
@@ -31,17 +31,36 @@ import (
 
 func (p *Server) generateEnv(req *http.Request) map[string]string {
 	scriptName := p.passthru
+
 	https := ""
 	if req.TLS != nil {
 		https = "On"
 	}
 
 	pathInfo := req.URL.Path
+	filenameMatch := false
+
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
 			scriptName = file
 			pathInfo = pathInfo[pos+4:]
+			filenameMatch = true
+		}
+
+	}
+
+	if filenameMatch == false && pathInfo != "/" && len(pathInfo) > 0 {
```

Comment:
```suggestion
	if len(pathInfo) > 1 {
```
seems sufficient here (once extracted to a function and thus not need `filenameMatch`)